dashboard/dashboard/pages/quality/columns.py

```diff
@@ -158,7 +158,7 @@
     "Admission": set(ADMISSION_COLUMNS),
     "Date": set(DATE_COLUMNS),
     "Demographic": set(DEMOGRAPHIC_COLUMNS),
-    "PHE": set(PMH_COLUMNS),
+    "PMH": set(PMH_COLUMNS),
```

Comment:
@taraganepola would it be useful spelling out "Previous medical history" here? (I mean I did need to search for PMH's meaning a while back, but not sure if NHS would have anyone not knowing that (and not sure if the space in the given dropdown allows for this.

Reply:
@imrehg , I had similar thoughts. I went with the acronym because of space issues. I'll have a quick play around with something shorter like "medical history" and see how that looks